modules/etcd-defrag.adoc

```diff
@@ -51,11 +51,16 @@ failed defrag on member: __<member_name>__, memberID: __<member_id>__: __<error_
 [id="manual-defrag-etcd-data_{context}"]
 == Manual defragmentation
 
-You can monitor the `etcd_db_total_size_in_bytes` metric to determine whether manual defragmentation is necessary.
+//You can monitor the `etcd_db_total_size_in_bytes` metric to determine whether manual defragmentation is necessary.
+
+A Prometheus alert indicates when you need to use manual defragmentation. The alert is displayed in two cases:
+
+   * When etcd uses more than 50% of its available space for more than 10 minutes
+   * When etcd is actively using less than 50% of its total database size for more than 10 minutes
```

Comment:
Should we have periods at the end of these?

Reply:
@stevsmit If I'm reading IBM Style [1] correctly, I don't think so. Both list items are sentence fragments, not complete sentences. 

[1] https://www.ibm.com/docs/en/ibm-style?topic=format-lists#punctuation-in-lists

Reply:
I figured it was because they were incomplete sentences. Thanks!